angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -170,6 +170,12 @@
                 <nimble-list-option *ngFor="let item of comboboxItems" [ngValue]="item">{{ item ? item.first : '' }}</nimble-list-option>
             </nimble-combobox>
         </div>
+        <div class="sub-container">
+            <div class="container-label">Rich Text Editor</div>
+            <div class="rich-text-editor-container">
+                <nimble-rich-text-editor placeholder="Rich text editor" #editor></nimble-rich-text-editor>
```

Comment:
Are we panning to add example `footer-actions` slot here, so that developers can learn what's the ideal way to use slots?

Reply:
I realized that we discussed this in one of the previous comments.